### PR TITLE
Pairing expiry reduction

### PR DESCRIPTION
**Pull request by llbartekll**
# Description
- removes Auth package
- pairing expiry reduction

Resolves # (issue)

## Due Dilligence

* [ ] Breaking change
* [ ] Requires a documentation update

### CODE REVIEW

File: Sources/WalletConnectSign/Auth/Services/Wallet/SessionAuthenticateResponder.swift [hunk 1]
```diff
@@ -141,11 +143,23 @@ actor SessionAuthenticateResponder {
...
     func respondError(requestId: RPCID) async throws {
+        let pairingTopic = try? util.getHistoryRecord(requestId: requestId).topic
         do {
-            try await walletErrorResponder.respondError(AuthError.userRejeted, requestId: requestId)
+            let _ = try await walletErrorResponder.respondError(AuthError.userRejeted, requestId: requestId)
+            Task {
+                if let pairingTopic = pairingTopic {
+                    removePairing(pairingTopic: pairingTopic)
+                }
+            }
```
Comment (jackpooleywc) [inline Sources/WalletConnectSign/Auth/Services/Wallet/SessionAuthenticateResponder.swift:153]: Maybe:
```suggestion
            if let pairingTopic = pairingTopic {
                 Task {
                    removePairing(pairingTopic: pairingTopic)
                }
            }
```

File: Sources/WalletConnectSign/Engine/Common/ApproveEngine.swift [hunk 2]
```diff
@@ -190,15 +188,21 @@ final class ApproveEngine {
             reason: reason
         )
 
-        if let pairingTopic = rpcHistory.get(recordId: payload.id)?.topic,
-           let pairing = pairingStore.getPairing(forTopic: pairingTopic),
-           !pairing.active {
-            pairingStore.delete(topic: pairingTopic)
+        // todo - delete and unsubscribe
```
Comment (jackpooleywc) [inline Sources/WalletConnectSign/Engine/Common/ApproveEngine.swift:191]: ?

**Review by jackpooleywc — COMMENTED** [review]
LGTM 👍

File: Example/ExampleApp.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -3277,8 +3265,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/WalletConnect/web3modal-swift";
 			requirement = {
-				kind = upToNextMajorVersion;
-				minimumVersion = 1.0.0;
+				branch = "pairing-expiry";
+				kind = branch;
```
[the remark this answers is not in the document]
Comment (llbartekll) [reply Example/ExampleApp.xcodeproj/project.pbxproj:3269]: yes, until we have a monorepo I need to reference to updated appkit

**Pull request merged**